Add Threat Intelligence Section (#7905)
opensearch-project/documentation-website: _security-analytics/threat-intelligence/index.md

---
layout: default
title: Threat intelligence
nav_order: 40
has_children: true
---

# Threat intelligence

Threat intelligence in Security Analytics offers the capability to integrate your threat intelligence feeds. Feeds are comprised Indicators of Compromise (IOCs), which search for malicious indicators within your data by setting up a threat intelligence monitor. These monitors generate findings and can send notifications when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.

You can interact with threat intelligence in the following ways:

> - Threat intelligence API: To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).

Author: ```suggestion
- Threat intelligence APsI: To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).
```

---
layout: default
title: Threat intelligence
nav_order: 40
has_children: true
---

# Threat intelligence

> You can use the threat intelligence feature in OpenSearch to integrate your threat intelligence feeds. Feeds are comprised of Indicators of Compromise (IoCs), which search for malicious indicators in your data. Through monitors, findings are generated, and notifications are sent when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.

Author: ```suggestion
You can use the threat intelligence feature in OpenSearch to integrate your threat intelligence feeds. Feeds are comprised of Indicators of Compromise (IOCs), which search for malicious indicators in your data. Through monitors, findings are generated, and notifications are sent when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.
```

> Threat intelligence in Security Analytics offers the capability to integrate your threat intelligence feeds. Feeds are comprised Indicators of Compromise (IoCs), which search for malicious indicators within your data by setting up a threat intelligence monitor. These monitors generate findings and can send notifications when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.

Reviewer: ```suggestion
You can use the threat intelligence feature in OpenSearch to integrate your threat intelligence feeds. Feeds are comprised of Indicators of Compromise (IoCs), which search for malicious indicators in your data. Through monitors, findings are generated, and notifications are sent when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.
```

> Threat intelligence in Security Analytics offers the capability to integrate your threat intelligence feeds. Feeds are comprised Indicators of Compromise (IOCs), which search for malicious indicators within your data by setting up a threat intelligence monitor. These monitors generate findings and can send notifications when malicious IPs, domains, or hashes referenced from the threat intelligence feeds match your data.

Reviewer: I had a different rewrite that looks like was overwritten?

Author: Let me look into that.

> To access threat intelligence, log into OpenSearch Dashboards, and select **Security Analytics** > **Threat Intelligence**.

Reviewer: I don't think this is relevant here. 
```suggestion

```

You can interact with threat intelligence in the following ways:

> - [Threat intelligence API]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/)

Reviewer: ```suggestion
- Threat intelligence API]. To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).
```

> - [Threat intelligence API]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).

Reviewer: ```suggestion
- [Threat intelligence API]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/)
```

> - [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/)

Reviewer: This seems a bit odd to link to the Getting Started topic. Perhaps we can do something like this, 
```suggestion
- OpenSearch Dashboards. To configure threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).
```

> - Threat intelligence API]. To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).

Reviewer: ```suggestion
- Threat intelligence API. To configure threat intelligence using API operations, see [Threat Intelligence APIs]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/api/threat-intel-api/).
```

> - OpenSearch Dashboards: To configure and use threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).

Reviewer: ```suggestion
- OpenSearch Dashboards. To configure and use threat intelligence through the OpenSearch Dashboards interface, see [Getting Started]({{site.url}}{{site.baseurl}}/security-analytics/threat-intelligence/getting-started/).
```

Reviewer: I would swap the two bullets above to match the order in the related articles section.

Author: I'll use the colon on the other one as well.
